en/docs/guides/login/oidc-token-encryption.md

```diff
@@ -45,84 +45,36 @@ Make the following changes to the created service provider.
 
 ## Try it
 
-This section guides you through obtaining an encrypted ID token and decrypting it using a simple java program. Alternatively, you can use the WSO2 IS playground sample application to decrypt the token. For instructions, see [Decrypt the ID token]({{base_path}}/guides/login/oidc-token-decryption).
+This section guides you through obtaining an encrypted ID token and decrypting it using the WSO2 IS playground sample application. Alternatively, you can use a simple java program to decrypt the token. For instructions, see [Decrypt the ID token]({{base_path}}/guides/login/oidc-token-decryption).
 
-1. See [OAuth Grant Types]({{base_path}}/access-delegation/authorization-code) and try out one of the grant types with the `openid` scope to obtain an access token.
+1. See [OAuth Grant Types]({{base_path}}/guides/access-delegation/authorization-code) and try out one of the grant types with the `openid` scope to obtain an access token.
 
 2. You will recieve an access token and an encrypted ID token. 
 
 3. To decrypt the ID token, provide the private key of the client. 
 
-    1.  Import JKS into a PKCS12 formatted store.
+    1. Import JKS into a PKCS12 formatted store.
 
         ``` java
         keytool -importkeystore -srckeystore testkeystore.jks -destkeystore testkeystore.p12 -srcstoretype JKS -deststoretype PKCS12 -srcstorepass wso2carbon -deststorepass wso2carbon -srcalias wso2carbon -destalias wso2carbon -srckeypass wso2carbon -destkeypass wso2carbon
         ```
 
-    2.  Extract the private key into a file named `key.pem`.
+    2. Extract the private key into a file named `key.pem`.
 
         ``` java
         openssl pkcs12 -in testkeystore.p12 -out key.pem -passin pass:wso2carbon -passout pass:wso2carbon -nodes -nocerts
         ```
 
-    3.  Open the created `key.pem` file using a text editor and you will see the extracted private key.
+    3. Open the created `key.pem` file using a text editor and you will see the extracted private key.
 
-    4.  Copy only the key string as shown in the sample below.
+    4. Copy only the key string as shown in the sample below.
 
         ![sample-key-string]({{base_path}}/assets/img/guides/sample-key-string.png)
+   
+    5. Paste the copied private key in the Client Private Key text area.
+   
+    6. Click **Decrypt** and the details of the decrypted ID Token will be displayed.
 
-4. The following sample JAVA program can be used to decrypt the ID token using the default `wso2carbon.jks` keystore. 
```

Comment:
Why was this removed?

Reply:
This was mentioned [here](https://is.docs.wso2.com/en/6.0.0/guides/login/oidc-token-decryption/) and this section only talks about [decrypting it using the WSO2 IS playground sample application](https://is.docs.wso2.com/en/6.0.0/guides/login/oidc-token-encryption/#try-it).